app/src/main/java/org/eyeseetea/malariacare/DashboardActivity.java

```diff
@@ -345,8 +371,14 @@ public void onClick(DialogInterface dialog, int arg1) {
         }
     }
 
+    public void closeSurveyFragmentAndAskReview(){
+        isLoadingReview=true;
+        initReview();
+    }
+
     public void closeSurveyFragment(){
         tabHost.getTabWidget().setVisibility(View.VISIBLE);
+        isLoadingReview=false;
```

Comment:
@idelcano Can you comment this method a little bit more? Otherwise in the future we won't have any idea what's going on here? Can you also use curly brackets for every if (in some cases you use it in some other you don't and it makes the code a bit unclear)? Is there any way we can do this if structures more clear?